scripts/scripts.js

```diff
@@ -38,7 +38,7 @@ window.hlx.plugins.add('performance', {
 
 window.hlx.plugins.add('experimentation', {
   condition: () => document.head.querySelector('[name^="experiment"],[name^="campaign-"],[name^="audience-"]')
-    || document.head.querySelector('[property^="campaign:-"],[property^="audience:-"]')
+    || document.head.querySelector('[property^="campaign:-"],[property^="audience:-"],[property^="campaign-"],[property^="audience-"]')
```

Comment:
Usually, it's `<meta property="campaign:-…">` or `<meta name="campaign-…">`, property without `:` is something I haven't encountered yet.

Reply:
Yes, you are right. I added that because I was thinking the user could omit the` :`, and declare as `Audience Desktop`. Then I realized it is better to have the norm that `: `should be required. I will revert this.

 For  `<meta property="campaign:-..>`, I am going to remove the hyphen here too, and only keep `[property^="campaign:"],[property^="audience:"]`. As the hyphen comes from the space, and should not considered as the condition.